src/assets/scss/application.scss

```diff
@@ -290,4 +290,59 @@ $nav-active-border-thickness: 5px;
   display: block;
   margin: govuk-spacing(2) 0;
   @include govuk-font($size: 48, $weight: bold);
-}
\ No newline at end of file
+}
+
+.language-select {
+  margin: govuk-spacing(2) 0;
+
+  &::after {
+    content: "";
+    display: block;
+    clear: both;
+  }
+
+  .language-select__list {
```

Comment:
Thanks for re-writing the classes here! Is there a reason why `.language-select__list` should be nested under `.language-select` in the scss? Is there a need for this higher level of specificity?

Reply:
We tend to avoid nesting unless necessary. The times we usually rely on nesting are with modifiers and on the rare occasion we might need to override some different rule so we need that level of specificity. 

In the `.account-navigation__link` example for instance, it has some rules that are specific for the "default" state of the link. But account nav links can also have an "active" state with slightly different styling hence the 
```
.account-navigation__link {
  ... default styles for .account-navigation__link...

  .account-navigation__list-item--active & {
    ...styles to apply "active" state to .account-navigation__link ...
  }
}
```
(not 100% sure why the active modifier was set to the link element's parent instead of the link itself in this particular instance, probably because there were tweaks in the styles of both elements for  the "active" state, but I hope the point makes sense anyway)

Reply:
It makes sense, yes. I have removed all nesting.